src/routes.tsx

```diff
@@ -17,55 +14,55 @@ const navItems: MenuItem[] = [
   {
     iconClass: 'fa pficon-topology',
     title: 'Graph',
-    to: kialiRoute('/service-graph/all'),
-    pathsActive: [/\/service-graph\/(.*)/]
+    to: '/service-graph/all',
+    pathsActive: [/^\/service-graph\/(.*)\//]
```

Comment:
Can you remove the `\/`?
It doesn't match an url like (notice after the namespace there is no `/`):
`http://localhost:3000/console/service-graph/kiali-test-breadth-0?layout=dagre&duration=60&edges=hide`

Reply:
Yes... ops...